pkg/server/manager/project.go

```diff
@@ -35,12 +35,13 @@ type ProjectManager interface {
 
 // projectManager represents the manager for project.
 type projectManager struct {
-	dataStore       *store.DataStore
-	pipelineManager PipelineManager
+	dataStore             *store.DataStore
+	pipelineManager       PipelineManager
+	pipelineRecordManager PipelineRecordManager
```

Comment:
`project` does not have direct relationship with `pipeline record`, so it does not need `pipelineRecordManager`.

Reply:
Oh, I forgot to delete it, I had fixed it.